packages/flutter/lib/src/widgets/list_wheel_scroll_view.dart

```diff
@@ -20,6 +21,175 @@ import 'scroll_position.dart';
 import 'scroll_position_with_single_context.dart';
 import 'scrollable.dart';
 
+/// A delegate that supplies children for [ListWheelScrollView].
+///
+/// [ListWheelScrollView] lazily construct its children during layout to avoid
+/// creating more children than are visible through the [Viewport], and this
```

Comment:
Suggestion: end the sentence at '[Viewport]' where the comma is now.

Reply:
fixed.